bamboo/select-stack.js

```diff
@@ -32,7 +32,7 @@ function determineIntegrationTestStackName(cb) {
     'Naga Nages': 'nnaga-ci',
     vpnguye2: 'vkn-ci',
     'Roger Kwarteng': 'rkwarten-ci',
-
+    'Paul Pilone': 'ppilone',
```

Comment:
Shouldn't this be for his CI stack? _Is this his CI stack_?

Reply:
@npauzenga  @paulpilone  you can call the stack whatever you want.      Note Jenny, Katie (!) and I don't have "-ci" - it's not required, the convention was just an opinionated thing that made it into an on-boarding doc at some point.       I'm not sure it's even that useful as we're rarely looking for ci vs dev stacks.     Open to other reasoning, but I don't think there's a reason it needs to be that.   

The only thing @paulpilone  needs to consider there is what he wants his dev stack to be named.

Reply:
Yeah, we can absolutely name stacks whatever we want. I _think_ though the `ppilone` stack is one that @paulpilone was deploying during planning. In other words, I think that was intended to be his "dev" stack and CI would create a new `ppilone-ci` stack. That was my assumption anyway.

You're right that it doesn't really matter but it might be nice to reserve `ppilone` as a stack to mess with and create a new `ppilone-ci` stack for... ci.